openedx/features/custom_fields/multiselect_with_other/db/fields.py

```diff
@@ -81,7 +83,8 @@ def validate(self, value, model_instance):
 
         :param value: list of all selected choice with other text.
         :param model_instance: current model instance for with it is saving data.
-        :return: None
+        :type value: list
+        :type model_instance: MultiSelectWithOtherField
```

Comment:
Our latest decision is to use GoogleStyle for docstrings which is also recommended and used by edX. Can we follow it from this PR to be consistent? If yes, Please update other docstrings as well.